understanding/20/three-flashes-or-below-threshold.html

```diff
@@ -38,12 +38,13 @@ <h2>Intent of Three Flashes or Below Threshold</h2>
          did not allow any flashing (even of a single pixel) within a broad frequency range
          (3 to 50 Hz). This Success Criterion is based on existing specifications in use in
          the UK and by others for television broadcast and has been adapted for computer display
-         viewing. The 1024 x 768 screen is used as the reference screen resolution for the
+         viewing. In WCAG 2.0, the 1024 x 768 screen was used as the reference screen resolution for the
          evaluation. The 341 x 256 pixel block represents a 10 degree viewport at a typical
          viewing distance. (The 10 degree field is taken from the original specifications and
          represents the central vision portion of the eye, where people are most susceptible
          to photo stimuli.)
       </p>
+      <p>With the proliferation of mobile and devices of many sizes, as well as the adoption of <a href="https://www.w3.org/TR/WCAG21/#dfn-css-pixels">CSS pixels</a> as a density-independent unit of measurement, the prior assessment criteria seem outdated. However, an image of a consistent size uses up relatively the same percentage of a user's visual field on any device. On a large screen, the image takes up less size, but the large screen takes up a larger part of the visual field. On a mobile screen, the image may take up most or all of the screen; however, the mobile screen itself takes up a smaller portion of the user's visual field. So the same dimension of the flashing content, represented in CSS pixels can still provide a consistent means of assessment. Substituting CSS pixels for the original pixel block means that the assessment becomes 341 x 256 CSS pixels, or a flashing area of 87,296 CSS pixels.</p>
```

Comment:
Nothing we can do about it, but perhaps worth mentioning that regardless of the ideal pixel font, what the specification itself (and content developers) can't account for is the actual viewing distance that a user chooses. Users that are closer to their screens than the idealised viewing distance will be affected even by flashing areas that normatively pass. The same problem also applies to users who rely on zoom/screen magnification. Conversely, users that are further away from the screen than the idealised viewing distance will tolerate flashing areas that are larger than the threshold.

Reply:
Not sure "mobile and devices of many sizes" sounds right. Perhaps just "With the proliferation of devices with varying screen sizes (from small-screen smartphones to very large desktop and living room displays" or similar?

Reply:
i do wonder if we could shave off that awkward looking single pixel and round it to 340 x 256 ... but i guess that would be a hard normative change too far

Reply:
latest commit addresses @patrickhlauke's comments